eng-paths/sr_software_engineer_3.md

```diff
@@ -0,0 +1,381 @@
+---
+title: Sr Software Engineer 3
+icon: fas fa-users
+order: 8
+layout: page-eng-paths
+---
+
+# Sr Software Engineer 3 (Sr Individual Contributor Path)
+### Years of xp (guideline)
+<table  markdown="1">
+<tr>
+<td>
+
+Sr Software Engineer 2
+
+</td>
+<td style="font-weight:bold">
+
+Sr Software Engineer 3
+
+</td>
+<td>
+
+Staff Engineer
+
+</td>
+</tr>
+<tr>
+<td  markdown="1">
+
+7+
+
+</td>
+<td  markdown="1" style="font-weight:bold">
+
+9+
+
+</td>
+<td  markdown="1">
+
+11+
+
+</td>
+</tr>
+</table>
+
+
+### Scope of action
+<table  markdown="1">
+<tr>
+<td>
+
+Sr Software Engineer 2
+
+</td>
+<td style="font-weight:bold">
+
+Sr Software Engineer 3
+
+</td>
+<td>
+
+Staff Engineer
+
+</td>
+</tr>
+<tr>
+<td  markdown="1">
+
+2-3 Products / Key components
+
+</td>
+<td  markdown="1" style="font-weight:bold">
+
+- Multiple Products/Key Components.
+- All team's domain and tasks that involve cross-teams work.
```

Comment:
This wording is hard to interpret. Are you trying to say the domains of all teams? Or all domains of their core team?

Reply:
I'd say all domains of their core team + cross-teams tasks, lemme fix it